app/controllers/agenda/minutes.js

```diff
@@ -227,10 +237,13 @@ export default class AgendaMinutesController extends Controller {
       );
 
       // *note: any changes made here should also be made in the minutes-report-generation service
-      const name = `Notulen - P${dateFormat(
+      let name = `VR PV ${dateFormat(
         this.meeting.plannedStart,
-        'yyyy-MM-dd'
-      )}`;
+        'yyyy'
+      )}/${this.meeting.number}`;
+      if (this.meeting.kind.get('uri') === constants.MEETING_KINDS.PVV) {
+        name += '-VV';
+      }
```

Comment:
Dit is de `numberRepresentation` van een meeting. Is het niet beter om gewoon dat te gebruiken? Dat geeft de gebruikers nog altijd de mogelijkheid om die te veranderen, wat kan in de edit meeting modal (alhoewel dat ik echt niet begrijp waarom, wanneer zouden ze dat manueel aanpassen dus waarom geven we zelfs die optie?)